gdk4/src/rgba.rs

```diff
@@ -71,22 +71,46 @@ impl RGBA {
         })
     }
 
+    // rustdoc-stripper-ignore-next
+    /// Creates a new builder-pattern struct instance to construct [`RGBA`] objects.
+    ///
+    /// This method returns an instance of [`RGBABuilder`] which can be used to create [`RGBA`] objects.
+    pub fn builder() -> RGBABuilder {
+        RGBABuilder::default()
+    }
+
     pub fn red(&self) -> f32 {
         self.0.red
     }
 
+    pub fn set_red(&self, red: f32) {
```

Comment:
```suggestion
    pub fn set_red(&mut self, red: f32) {
```

Also for the others :scream: